bakery/baking_usage_examples/RB_1_qubit/RB_utils.py

```diff
@@ -0,0 +1,110 @@
+import pandas as pd
+from bakery.bakery import *
+from qm.qua import *
+
+c1_ops = [  # Clifford operations
+    ('I',),
+    ('X',),
+    ('Y',),
+    ('Y', 'X'),
+
+    ('X/2', 'Y/2'),
+    ('X/2', '-Y/2'),
+    ('-X/2', 'Y/2'),
+    ('-X/2', '-Y/2'),
+    ('Y/2', 'X/2'),
+    ('Y/2', '-X/2'),
+    ('-Y/2', 'X/2'),
+    ('-Y/2', '-X/2'),
+
+    ('X/2',),
+    ('-X/2',),
+    ('Y/2',),
+    ('-Y/2',),
+    ('-X/2', 'Y/2', 'X/2'),
+    ('-X/2', '-Y/2', 'X/2'),
+
+    ('X', 'Y/2'),
+    ('X', '-Y/2'),
+    ('Y', 'X/2'),
+    ('Y', '-X/2'),
+    ('X/2', 'Y/2', 'X/2'),
+    ('-X/2', 'Y/2', '-X/2'),
+
+]
+
+# Cayley table corresponding to above Clifford group structure
+
+c1_table = pd.read_csv('c1_cayley_table.csv').to_numpy()[:, 1:]
+
+
+class RB_one_qubit:
+    def __init__(self, config: dict, d_max: int, K: int, qubit: str, cliffords: List = None,
```

Comment:
not sure I understand how cliffords can be a list. What is the meaning of every elementn in the list? i.e how do I know that entry zero corresponds to 'X', entry 1 to 'Y' and so on? I think it should be a dict no? e.g:
```
RBOneQubit(..., cliffords = {'X': 'my_x', 'Y': 'my_y', ....} )
```

Reply:
cliffords should be a list whose index is related to which index should we look for in the provided Cayley table (we move from one Clifford to another, keeping track on the change using indices). For now it is true that we could move forward with a dictionary, but in the hope of moving everything to QUA later I do think we shall rely on this moving related to indices. Main issue with dict is that to my knowledge it has no numerical ordering (unless we do an enumerate but then we are coming back to a list).